bindata/v4.1.0/kube-apiserver/boundsatokenconfig.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: kubecontrolplane.config.openshift.io/v1
+kind: KubeAPIServerConfig
+apiServerArguments:
+  # The following arguments are required to enable bound sa
+  # tokens. This is only supported post-bootstrap so these
+  # values must not appear in defaultconfig.yaml.
+  service-account-issuer:
+  - auth.openshift.io
+  api-audiences:
+  - auth.openshift.io
+  service-account-signing-key-file:
+  - /etc/kubernetes/static-pod-certs/secrets/bound-service-account-signing-key/service-account.key
+serviceAccountPublicKeyFiles:
+  - /etc/kubernetes/static-pod-resources/configmaps/sa-token-signing-certs
+  # The following path contains the public keys needed to verify bound sa
+  # tokens. This is only supported post-bootstrap.
+  - /etc/kubernetes/static-pod-resources/configmaps/bound-sa-token-signing-certs
```

Comment:
just copy these into the overrides file

Reply:
Which overrides file?  afaict the only ones available are for bootstrap.  Would you prefer that I add another post-bootstrap override file rather than adding this feature-specific one?

Reply:
https://github.com/openshift/cluster-kube-apiserver-operator/blob/master/bindata/bootkube/config/config-overrides.yaml

Reply:
This file has been renamed to config-overrides.yaml as requested.